packages/ccda/src/ccda-to-fhir.ts

```diff
@@ -151,6 +1273,50 @@ class CcdaToFhirConverter {
     this.processHeader();
     const composition = this.createComposition();
 
+    // If QRDA generation is requested, build QRDA document
+    if (this.options?.generateQRDA && this.options.qrdaParams && this.qrdaPatientData) {
```

Comment:
No, this should be handled with `options.type === 'qrda'`, not an entirely new code path.